docs/index.rst

```diff
@@ -10,19 +13,19 @@ OpenQL is a framework for high-level quantum programming in C++/Python. The fram
 
    installation
    start
-   changelog
-   contributors
 
 .. toctree::
    :maxdepth: 3
    :caption: OpenQL Basics:
```

Comment:
I think the ordering in this table of contents can be improved a bit. 

Right now the user is inclined to click on the "getting started" link but that only contains an example and no explanation. 
I think the documentation right now misses a "getting started" that explains on a higher level what the core concepts are. This is done in the beginning of the "quantum_gate" section but it is not clear to a novice that that is where one should look. 

![Screenshot 2019-12-10 at 20 04 54](https://user-images.githubusercontent.com/6142932/70560136-57c12780-1b88-11ea-91df-c28d381ff0a7.png)